mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -265,8 +272,8 @@ pub fn p2p_private_and_peer_id_to_broadcast(ctx: &MmArc, p2p_privkey: Option<&Ke
 /// to stop it
 pub fn broadcast_swap_msg_every<T: 'static + Serialize + Clone + Send>(
     ctx: MmArc,
-    topic: String,
-    msg: T,
+    topic: String, // topic
+    msg: T,        // message
```

Comment:
These comments actually just repeat fields names.
If you dont need more comprehensive comments here, please remove these comments.

Reply:
removed